extensions/sql-migration/package.json

```diff
@@ -178,7 +178,8 @@
     "@microsoft/ads-service-downloader": "^1.2.1",
     "@microsoft/ads-extension-telemetry": "^3.0.1",
     "uuid": "^8.3.2",
-    "vscode-nls": "^4.1.2"
+    "vscode-nls": "^4.1.2",
+    "@azure/storage-blob": "^12.23.0-beta.1"
```

Comment:
Have tested for any regression, as we are using Storage account for read the blob many other places well. Just test for a complete File share migration, and a blob migration.

Reply:
at other places the utility functions related to storage account are referred from azure core extension, the module that I added is for my change only.